src/mono/mono/metadata/sre.c

```diff
@@ -4685,20 +4666,6 @@ ves_icall_ModuleBuilder_getMethodToken (MonoReflectionModuleBuilderHandle mb,
 	return mono_image_create_method_token (MONO_HANDLE_GETVAL (mb, dynamic_image), MONO_HANDLE_CAST (MonoObject, method), opt_param_types, error);
 }
 
-#ifndef ENABLE_NETCORE
-void
-ves_icall_ModuleBuilder_WriteToFile (MonoReflectionModuleBuilderHandle mb, HANDLE file, MonoError* error)
-{
-	mono_image_create_pefile (MONO_HANDLE_RAW (mb), file, error);
-}
-
-void
-ves_icall_ModuleBuilder_build_metadata (MonoReflectionModuleBuilderHandle mb, MonoError* error)
-{
-	mono_image_build_metadata (MONO_HANDLE_RAW (mb), error);
-}
-#endif
-
```

Comment:
this will conflict with https://github.com/dotnet/runtime/pull/47915 :)

Reply:
That's ok, I'll rebase mine on top of this